productivetools.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Tools
+permalink: /tools/
+---
```

Comment:
It might make sense to replace the title "Tools" with another term. My suggestion is "Unsere Projekte". Nevertheless "Tools" would also be okay for me. It's just a spontaneous suggestion because I had not had the idea to click on "Tools" when I first looked at the website.

Reply:
I agree :-)

Reply:
My goal is to separate Tools which everyone can use online without any tech knowhow and tools, which are still in development. All of them are "Projekte"... I know "Tools" is still not optimal, even when the next menu title would be "In Entwicklung" oder similar... at this moment, I have no idea for better wording.

Reply:
If you guys get struck by a good idea for better wording... shout out! :)

Reply:
How about

- Live Tools
- Nerd Tools (and as title - on mouse-hover - something like "you need some engineering to handle these tools")
- In Progress

Reply:
Maybe we can call the menu items "Webtools" and "In Entwicklung". Webtools for all the released web-based tools and "In Entwicklung" for ongoing, not yet released/published projects.

Reply:
ah I missunderstood the first comment of @ThomasHaefliger...

- Webtools
- In Entwicklung

as @wp99cp has proposed.

Reply:
Thx! It will be 

- Webtools
- In Entwicklung

as proposed.
I'll change it and merge

Reply:
Can one of you explain to me how to sort the menu items? I want first Webtools, then In Entwicklung.
I added a weight attribute to the pages, tried [this](https://jekyllrb.com/tutorials/navigation/#scenario-2-sorted-list) and [that](https://stackoverflow.com/questions/9053066/sorted-navigation-menu-with-jekyll-and-liquid) but no success.
Still learning :)